src/modules/admin/components/forms/FormDefinitionTable.tsx

```diff
@@ -49,24 +49,26 @@ const columns: ColumnDef<Row>[] = [
 ];
```

Comment:
I haven't made these updates yet, but my understanding is that in-scope for this ticket are all updates to make our forms table match the figma design here: https://www.figma.com/file/F9EHR2IZYuMAlOJ9aOG2IL/OP-%2F-HMIS-%2F-2024-%2F-Q1-%2B-Q2-(Jan-1---June-30)?type=design&node-id=2611-31087&mode=design&t=MS7446xMjNXDLuZX-0
including adding new columns and changing some column names around. Is that correct?